src/DotNetEnv/Env.cs

```diff
@@ -8,7 +8,8 @@ public class Env
             string path,
             bool trimWhitespace = true,
             bool isEmbeddedHashComment = true,
-            bool unescapeQuotedValues = true
+            bool unescapeQuotedValues = true,
+            bool clobber = true
```

Comment:
How would you feel about changing the name to `clobberExistingVars` to be more explicit?